index.html

```diff
@@ -3,314 +3,423 @@
+      border-radius: 24px;
+    }
+    .btn-group .btn:active{
+      background: #FD8D3C;
+      color: #FFFFFF;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline:0;
+    }
+    .btn:focus{
+      background-color: #FD8D3C;
+      color: #4C4C4C;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline: 0
+    }
+    .btn.active{
+      background-color: #FD8D3C;
+      color: #4C4C4C;
+      border: 2px solid #FFFFB2;
+      border-radius: 24px;
+      outline:0;
+    }
+    
+
+
+    a{color: #FFFFB2;}
+    a:hover{color: #FD8D3C;}
+
+    td:nth-child(1),th:nth-child(1) { width:200px; }
+    td:nth-child(2),th:nth-child(2) { text-align: center; }
+    td:nth-child(4),th:nth-child(4) { text-align: center; }
+    
+    tr:hover td{font-weight: bold;
+      color: #FFFFFF;
     }
-  </style>
+
+    body {
+      background-color: #4C4C4C;
+      font-family: 'Arimo', sans-serif;
+      color: #CECECE;
+          }
+
+    th {
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+        width: 400px;
+                } 
+    h1,h3,h5,h2{
+        font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+                }   
+     </style>
 </head>
 <body>
-<div class="header section">
   <div class="container">
-    <h1>Week 2: Javascript and the DOM</h1>
-    <p>Let's wire up some buttons.</p>
-  </div>
-</div>
-<div class="controls section">
-  <div class="container">
-    <div class="btn-toolbar">
-      <div class="btn-group">
-        <button type="button" class="btn btn-default" id="sort-asc">Ascending</button>
-        <button type="button" class="btn btn-default" id="sort-desc">Descending</button>
-      </div>
-      <div class="btn-group">
-        <button type="button" class="btn btn-default" id="filter-es">Elementary</button>
-        <button type="button" class="btn btn-default" id="filter-ms">Middle</button>
-        <button type="button" class="btn btn-default" id="filter-hs">High</button>
-      </div>
+    <div class="header">
+      <h1>Felony Disenfranchisement</h1>
+      <p>The number of people disenfranchised due to a felony conviction has
+escalated dramatically in recent decades as the population under criminal
+justice supervision has increased. There were an estimated 1.17 million
+people disenfranchised in 1976, 3.34 million in 1996, and over 5.85 million in
+2010. The table below gives all states and their rates of felon disenfranchisement in 2010, which is defined as the total percentage of the voting age population disenfranchised due to a felony conviction.</p>
+
     </div>
-  </div>
-</div>
-<div class="main section">
-  <div class="container">
-    <table class="table" id="data-table">
+  <div class ="controls section">
+    <div class="container">
+      
+      <h4>Filter</h4>
+        <div class="btn-group" role="group" aria-label="Filter">
+          <button type="button" class="btn btn-default" id="filter-No">No Restrictions</button>
```

Comment:
Purely a style consideration, but I prefer IDs and classes to be all lowercase, words separated by hyphens: `filter-no`. Whatever you choose to do, the important thing is to be consistent, as inconsistency leads to errors and lots of "wait, what did I put there?"